package-lock.json

```diff
@@ -12257,21 +11194,18 @@
         },
         "node_modules/json-schema-traverse": {
             "version": "0.4.1",
-            "resolved": "https://registry.npmjs.org/json-schema-traverse/-/json-schema-traverse-0.4.1.tgz",
-            "integrity": "sha512-xbbCH5dCYU5T8LcEhhuh7HJ88HXuW3qsI3Y0zOZFKfZEHcpWiHU/Jxzk629Brsab/mMiHQti9wMP+845RPe3Vg==",
-            "dev": true
+            "dev": true,
```

Comment:
resolved/integrity fields were removed for all items. Could try to avoid that: https://stackoverflow.com/a/75236714/152142

Perhaps it's not worth the trouble, but it will cause some churn.

More generally: when package-lock.json has massive changes, that's worth double-checking and note whether it's expected, in your PR description.

Reply:
Just verified and the dependencies can be updated so that we do not have this big `package-lock.json` diff

Reply:
Unclear to me why that happened but fixed in latest version.